.zap/rules.conf

```diff
@@ -1,9 +1,14 @@
-# Mark the following rules as INFO
+# Mark the following rules as IGNORE
```

Comment:
the PR title and description are ambiguous - should specify it's updating zap rules

Reply:
updated